canonical-web-and-design/ubuntu.com
# Add redirect of '/engage/iot-elkhart-lake' to /download/iot/intel-iotg'

## Done

- Add redirect of '/engage/iot-elkhart-lake' to /download/iot/intel-iotg' to the redirects.yaml

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check the redirect works

## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/10848

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)
